packages/format-library/package.json

```diff
@@ -31,7 +31,10 @@
 		"@wordpress/block-editor": "file:../block-editor",
 		"@wordpress/components": "file:../components",
 		"@wordpress/compose": "file:../compose",
+		"@wordpress/core-data": "file:../core-data",
 		"@wordpress/data": "file:../data",
+		"@wordpress/date": "file:../date",
+		"@wordpress/editor": "file:../editor",
```

Comment:
We shouldn't be adding all these dependencies to this package. But this should be resolved if we move away from adding this control as a format.